src/pages/about/aboutHelper.js

```diff
@@ -3,142 +3,327 @@
  * @type {[{image: string, fullName: string, title: string}, {}]}
  */
 export const teamImages = [
```

Comment:
If this array is no longer being used, because the information is now spread out in the team specific arrays, it can be deleted